includes/admin/settings/class-settings-email.php

```diff
@@ -163,6 +163,24 @@ public function email_notification_setting() {
 			$email_notifications_table->prepare_items();
 			$email_notifications_table->display();
 		}
+
+		/**
+		 * Output the settings.
+		 *
+		 * Note: if you want to overwrite this function then manage show/hide save button in your class.
+		 *
+		 * @since  1.8
+		 * @return void
+		 */
+		public function output() {
+			if ( $this->enable_save ) {
+				$GLOBALS['give_hide_save_button'] = apply_filters( 'hide_save_button_on_email_page', false );
```

Comment:
@Sidsector9 please make sure you always add `give` as a prefix.

Change filter name to `give_hide_save_button_on_email_admin_setting_page`